frontends/election-manager/src/utils/cast_vote_record_files.ts

```diff
@@ -188,6 +201,77 @@ export class CastVoteRecordFiles {
 
     return result;
   }
```

Comment:
It seems like `addAllFromFileSystemEntries` isn't used anymore, and `addFromFileSystemEntry` is only called by this function. Can all that code be removed?

Reply:
Similarly, `add` and `addAll` are only used in one place in the modal to add a single file (besides being used in tests), so they might be candidates for removal as well

(I started looking into this because I was getting confused trying to understand the interface of this module, and realized it has a lot of entry points and could probably be reduced to a few key functions)